partner_compassion/models/partner_segment.py

```diff
@@ -0,0 +1,53 @@
+##############################################################################
+#
+#    Copyright (C) 2021 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Jonathan Guerne <jonathan.guerne@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import fields, models
+
+
+class ResPartnerSegment(models.Model):
+    _name = "res.partner.segment"
+    _description = "Partner Segmentation"
+
+    segment_index = fields.Integer(string='Segment Index used for segment computation by segmentation engine')
```

Comment:
- String is usually the label in the view so you must keep it short. Use the help parameter to give more insight.
- String is the first attribute of field and can be omitted : in that case the label will be as the field_name => Field Name
- Add some other parameters
- Lines should be 89 characters long at most
```suggestion
    segment_index = fields.Integer(
        required=True,
        help="Used for segment computation by segmentation engine"
    )
```